src/App.tsx

```diff
@@ -1,16 +1,30 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
-import { Loader } from './components/Loader';
+import { TodoContext } from './Contexts/TodoContext';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [filterBy, setFilterBy] = useState('all');
+  const [searchQuery, setSearchQuery] = useState('');
+  const [currentTodo, setCurrentTodo] = useState<Todo | null>(null);
```

Comment:
Not critical, but sometimes it is better to 
1. Encapsulate these states in the context
2. Delegate loading/filtering TODOs to the context too